addons/sale/models/account_move_line.py

```diff
@@ -151,23 +145,7 @@ def _sale_determine_order(self):
         """ Get the mapping of move.line with the sale.order record on which its analytic entries should be reinvoiced
             :return a dict where key is the move line id, and value is sale.order record (or None).
         """
-        mapping = {}
-        for move_line in self:
-            if move_line.analytic_distribution:
-                distribution_json = move_line.analytic_distribution
-                account_ids = [int(account_id) for key in distribution_json.keys() for account_id in key.split(',')]
-
-                sale_order = self.env['sale.order'].search([('analytic_account_id', 'in', account_ids),
-                                                            ('state', '=', 'sale')], order='create_date ASC', limit=1)
-                if sale_order:
-                    mapping[move_line.id] = sale_order
-                else:
-                    sale_order = self.env['sale.order'].search([('analytic_account_id', 'in', account_ids)],
-                                                               order='create_date ASC', limit=1)
-                    mapping[move_line.id] = sale_order
-
-        # map of AAL index with the SO on which it needs to be reinvoiced. Maybe be None if no SO found
-        return mapping
+        return {}
```

Comment:
So this doesn't work without project anymore?

Reply:
No but it's an acceptable regression after discussion with the PO of accounting (labe)